test/ad/distributions.jl

```diff
@@ -307,6 +314,7 @@
         DistSpec((df, A) -> InverseWishart(df, to_posdef(A)), (3.0, A), B, to_posdef),
         DistSpec((df, A) -> TuringWishart(df, to_posdef(A)), (3.0, A), B, to_posdef),
         DistSpec((df, A) -> TuringInverseWishart(df, to_posdef(A)), (3.0, A), B, to_posdef),
+        DistSpec(() -> LKJ(3, 1.), (), A, to_corr), # AD for parameters of LKJ requires more DistributionsAD supports
```

Comment:
Can you add it just to the broken distributions? We should make sure to open an issue at DistributionsAD if it's not working, I guess it's difficult to keep track of it if it's just mentioned in the code.

Reply:
Added broken test.